dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/dataset/JdbcCompleteDataSetRegistrationExchangeStore.java

```diff
@@ -93,6 +93,8 @@
     
     private static final String P_COMPLETED = "completed";
 
+    private static final String P_COMPLETED = "completed";
+
```

Comment:
Are these two lines identical?

Reply:
@larshelge  closing this PR as this was somehow  fixed for 2.32 but not for other versions.